golem_messages/message/tasks.py

```diff
@@ -234,25 +234,54 @@ def __init__(self, *args, **kwargs):
 
 @library.register(TASK_MSG_BASE + 1)
 class WantToComputeTask(ConcentEnabled, base.Message):
+    """
+    A computation Offer.
+
+    A Provider sends it directly to a Requestor as a response to the Requestor's
+    Demand (Task) in order to get work (SubTask ie. TaskToCompute) to do.
+
+    Attributes:
+        node_name:          Provider's node name
+        perf_index:         Provider's Performance index
+        max_resource_size:  Provider's Storage size available for computation
+        max_memory_size:    Provider's RAM
+        num_cores:          Provider's CPU cores (deprecated)
+        price:              Offered price in GNT "WEI" (10e-18)
+        num_subtasks:       How many subtasks Provider wants to work on
+                            (simultaneously); 1 by default
+        concent_enabled:    Provider's Concent status
+        extra_data:         additional required information about the Provider's
+                            environment. `golem-messages` should be
+                            intentionally agnostic with regards to the contents
+                            of this field.
+        provider_public_key:
+                            for signing and encryption
+        provider_ethereum_public_key:
+                            for transactions on ETH blockchain
+        task_header:        Demand; signed by a Requestor
+    """
     __slots__ = [
         'node_name',
         'perf_index',
         'max_resource_size',
         'max_memory_size',
         'num_cores',
         'price',
-        'concent_enabled',  # Provider notifies requestor
-                            # about his concent status
+        'num_subtasks',
+        'concent_enabled',
+        'extra_data',
+        # TODO: `provider` prefix is redundant; all above fields refers Provider
```

Comment:
But it's consistent with `TaskToCompute.requestor_public_key`

Reply:
I was trying to understand this naming convention. To be consistent all fields relative to requestor or provider should have such prefixes. Furthermore TTC `requestor_id` is redundant bc it is also in `ttc.wtct.th.task_owner.key` isn't it?

It is too much for me now. I'd stop here. Do you want me to remove this TODO, or change its sound?

Reply:
Replying to https://github.com/golemfactory/golem-messages/pull/335/files#r276157058

Yes, if there is no issue yet ( and the todo stays ) please make one so its not only in code but also on a backlog

Reply:
If it's important and worth the effort, than please create an issue and link it in comment. Otherwise pls remove this TODO.